src/lager_elogfmt_formatter.erl

```diff
@@ -21,10 +21,11 @@ format(Msg, Config, _Color) ->
 format(Msg, Config) ->
     App = proplists:get_value(app, Config),
     Defaults = proplists:get_value(defaults, Config, []),
+    StripPid = proplists:get_value(strip_pid, Config, true),
     Props = filter_undefined([{"app", App},
                               severity(Msg) |
                               Defaults ++
-                              meta(Msg, App)
+                              meta(Msg, App, StripPid)
```

Comment:
I'm not super happy about passing the boolean around. Why not add the pid to meta either way and filter it out later?